examples/simple_static_task/static_test_script.py

```diff
@@ -19,9 +19,9 @@
 TASK_DIRECTORY = os.path.join(get_root_dir(), "examples/simple_static_task")
 
 defaults = [
-    {"mephisto.blueprint": BLUEPRINT_TYPE},
-    {"mephisto.architect": 'local'},
-    {"mephisto.provider": 'mock'},
+    {"mephisto/blueprint": BLUEPRINT_TYPE},
+    {"mephisto/architect": 'local'},
+    {"mephisto/provider": 'mock'},
```

Comment:
If you have any corresponding yaml files, you should update their directory structure as well.

Reply:
All of these are structured configs only.